src/SentTimerTeensyQuad.h

```diff
@@ -0,0 +1,163 @@
+#pragma once
+
+#include "sent/BaseSent.h"
+#include "imxrt.h"
+
+const unsigned char numPins = 16;
+
+const unsigned char validPins[numPins] = {
+    10, 12,  1, 13, 40, 41, 19, 42, 18, 43, 14, 44, 15, 45, 6,  9,
+};
+
+const unsigned char tmrIndex[numPins] = {
+    1, 1, 1, 2, 2, 2, 3, 3, 3, 3, 3, 3, 3, 4, 4, 4,
+};
+
+const unsigned char chnIndex[numPins] = {
+    0, 1, 2, 0, 1, 2, 0, 0, 1, 1, 2, 2, 3, 0, 1, 2,
+};
+
+volatile uint32_t *ioMuxCtrl[numPins] = {
+    &IOMUXC_SW_MUX_CTL_PAD_GPIO_B0_00,
+    &IOMUXC_SW_MUX_CTL_PAD_GPIO_B0_01,
+    &IOMUXC_SW_MUX_CTL_PAD_GPIO_B0_02,
+    &IOMUXC_SW_MUX_CTL_PAD_GPIO_B0_03,
+    &IOMUXC_SW_MUX_CTL_PAD_GPIO_B0_04,
+    &IOMUXC_SW_MUX_CTL_PAD_GPIO_B0_05,
+    &IOMUXC_SW_MUX_CTL_PAD_GPIO_AD_B1_00,
+    &IOMUXC_SW_MUX_CTL_PAD_GPIO_B0_06,
+    &IOMUXC_SW_MUX_CTL_PAD_GPIO_AD_B1_01,
+    &IOMUXC_SW_MUX_CTL_PAD_GPIO_B0_07,
+    &IOMUXC_SW_MUX_CTL_PAD_GPIO_AD_B1_02,
+    &IOMUXC_SW_MUX_CTL_PAD_GPIO_B0_08,
+    &IOMUXC_SW_MUX_CTL_PAD_GPIO_AD_B1_03,
+    &IOMUXC_SW_MUX_CTL_PAD_GPIO_B0_09,
+    &IOMUXC_SW_MUX_CTL_PAD_GPIO_B0_10,
+    &IOMUXC_SW_MUX_CTL_PAD_GPIO_B0_11,
+};
+
+unsigned char ioMuxValue[numPins] {
+    1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1,
+};
+
+volatile uint32_t *ioSelCtrl[numPins] = {
+    nullptr,
+    nullptr,
+    nullptr,
+    &IOMUXC_QTIMER2_TIMER0_SELECT_INPUT,
+    &IOMUXC_QTIMER2_TIMER1_SELECT_INPUT,
+    &IOMUXC_QTIMER2_TIMER2_SELECT_INPUT,
+    &IOMUXC_QTIMER3_TIMER0_SELECT_INPUT,
+    &IOMUXC_QTIMER3_TIMER0_SELECT_INPUT,
+    &IOMUXC_QTIMER3_TIMER1_SELECT_INPUT,
+    &IOMUXC_QTIMER3_TIMER1_SELECT_INPUT,
+    &IOMUXC_QTIMER3_TIMER2_SELECT_INPUT,
+    &IOMUXC_QTIMER3_TIMER2_SELECT_INPUT,
+    &IOMUXC_QTIMER3_TIMER3_SELECT_INPUT,
+    nullptr,
+    nullptr,
+    nullptr,
+};
+
+unsigned char ioSelValue[numPins] {
+    0, 0, 0, 1, 1, 1, 1, 2, 0, 2, 1, 2, 1, 0, 0, 0,
+};
+
+volatile IMXRT_TMR_t *quadTimers[4] = {
+    (IMXRT_TMR_t *)IMXRT_TMR1_ADDRESS, 
+    (IMXRT_TMR_t *)IMXRT_TMR2_ADDRESS, 
+    (IMXRT_TMR_t *)IMXRT_TMR3_ADDRESS, 
+    (IMXRT_TMR_t *)IMXRT_TMR4_ADDRESS
+};
+
+void isr_tmr1();
+void isr_tmr2();
+void isr_tmr3();
+void isr_tmr4();
+
+void (*isrFuncs[4])() = {isr_tmr1, isr_tmr2, isr_tmr3, isr_tmr4};
+IRQ_NUMBER_t irqNumbers[4] = {IRQ_QTIMER1, IRQ_QTIMER2, IRQ_QTIMER3, IRQ_QTIMER4};
+void *listeners[4][4] = {nullptr};
```

Comment:
4? How about numTimers and numChannels?

Reply:
What's the harm in a little magic number in our Arduino code?
Will fix.